Engine/BaseAssets/Components/SoundListener.cs

```diff
@@ -13,18 +12,23 @@ public bool IsCurrent
             set
             {
                 if (value)
-                    makeCurrent();
+                    MakeCurrent();
                 else
                 {
                     if (IsCurrent)
-                        SoundCore.CurrentListener = this;
+                        SoundCore.CurrentListener = null;
                 }
             }
         }
 
-        public SoundListener()
+        protected override void OnInitialized()
         {
             Listener = new Listener();
+
+            if (SoundCore.CurrentListener == null) //TODO: Unity allows only 1 AudioListener in the scene. Otherwise throws warning
```

Comment:
we allow multiple existing, but only one active

Reply:
++